bin/blocks-plot

```diff
@@ -34,14 +34,14 @@ def plot_dataframe(dataframe):
     print("Plotting {} channels:".format(len(dataframe.columns)))
     for cname, series in iteritems(dataframe):
         print("    {}".format(cname))
-        pylab.plot(t, series.interpolate(), label=cname)
```

Comment:
Yes. 

Ok, I see your point! And this is indeed a problem:

If we monitor two quantities with different frequencies (e.g. a_cost every epoch, and b_cost only every 2nd) .. than we can't plot them into the same figure if the interpolate() is missing.

I'll look for a better solution -- something closer to: interpolating in the middle but not interpolating from the right most NaN to the end.